workspaces/adr/plugins/adr-backend/src/service/router.ts

```diff
@@ -18,7 +18,7 @@ import { CacheClient, UrlReader } from '@backstage/backend-common';
 import { NotModifiedError, stringifyError } from '@backstage/errors';
 import express from 'express';
 import Router from 'express-promise-router';
-import { madrParser } from '../search/madrParser';
+import { madrParser } from '@backstage-community/search-backend-module-adr';
```

Comment:
this module isn't even a dependency. I think this used to be correct.

```suggestion
import { madrParser } from '../search/madrParser';
```

Reply:
ah good catch! I believe this actually should be moved to the `adr-common` package so it can be shared among the search and backend packages

Reply:
I've moved a bunch of types to the common package, under `src/search.ts`, this should be clearer now and give people the option to use these types now